rush.json

```diff
@@ -65,19 +65,6 @@
    */
   // "suppressNodeLtsWarning": false,
 
-  /**
-   * If you would like the version specifiers for your dependencies to be consistent, then
-   * uncomment this line. This is effectively similar to running "rush check" before any
-   * of the following commands:
-   *
-   *   rush install, rush update, rush link, rush version, rush publish
-   *
-   * In some cases you may want this turned on, but need to allow certain packages to use a different
-   * version. In those cases, you will need to add an entry to the "allowedAlternativeVersions"
-   * section of the common-versions.json.
-   */
-  "ensureConsistentVersions": true,
```

Comment:
This should only change after Rush has been published with this new feature and has been bumped in this repo.

Reply:
Bumped rush version in this MR